pyproject.toml

```diff
@@ -27,6 +27,7 @@ PyYAML = "^6.0"
 anndata = "^0.7.8"
 scikit-learn = "^1.0.1"
 switchlang = "^0.1.0"
+scanpy = "^1.8.2"
```

Comment:
Think we should make many of the dependencies more loose and work with `>=` instead of '^='. 
But we can do that in a follow up PR. Or I can do that :)

Reply:
Maybe you can create an issue for that @antschum ?